package.json

```diff
@@ -6,19 +6,21 @@
   "engines": {
     "node": ">=12.0.0 <17.0.0"
   },
+  "bin": {
+    "govuk-prototype-kit": "scripts/cli"
+  },
   "scripts": {
-    "start": "node start",
-    "build": "node lib/build/generate-assets",
-    "serve": "node listen-on-port",
+    "start": "echo 'there is no start script, honest.'",
```

Comment:
Why is this necessary?

Reply:
Without a no-op `start` it tries to run `server.js` which doesn't work because there's no kit.